docs/verify/index.mdx

```diff
@@ -19,7 +19,7 @@ For general quantum circuits, the following tools are available to verify your q
 
 - For simulating circuits when using Qiskit Runtime, use its [local testing mode.](local-testing-mode)
 - For exact simulation of small quantum circuits, you can use the reference primitives included with Qiskit. See [Exact simulation with Qiskit primitives](simulate-with-qiskit-primitives).
-- For higher-performance simulation that can handle larger circuits, or to incorporate noise models into your simulation, use [Qiskit Aer](https://qiskit.org/ecosystem/aer/), a project that is part of the [Qiskit Ecosystem](https://qiskit.github.io/ecosystem/). See [Exact and noisy simulation with Qiskit Aer primitives](simulate-with-qiskit-aer) (V1 primitive support only). For V2 primitive support, use the [Qiskit Runtime local testing mode](local-testing-mode) with a Qiskit Aer backend.
+- For higher-performance simulation that can handle larger circuits, or to incorporate noise models into your simulation, use [Qiskit Aer](https://qiskit.org/ecosystem/aer/), a project that is part of the [Qiskit Ecosystem](https://qiskit.github.io/ecosystem/). See [Exact and noisy simulation with Qiskit Aer primitives](simulate-with-qiskit-aer) (V1 primitive support only).
```

Comment:
This rewrite doesn't make it clear that V2 primitives _are_ supported. Should it be this?

```suggestion
- For higher-performance simulation that can handle larger circuits, or to incorporate noise models into your simulation, use [Qiskit Aer](https://qiskit.org/ecosystem/aer/), a project that is part of the [Qiskit Ecosystem](https://qiskit.github.io/ecosystem/). See [Exact and noisy simulation with Qiskit Aer primitives](simulate-with-qiskit-aer).
```

Reply:
Yes; good catch!